packages/requestNetwork.js/test/deploy.ts

```diff
@@ -77,6 +87,78 @@ web3Single.getDefaultAccount().then((creator) => {
                 addressRequestEthereum = newContractInstance.options.address;
                 newContractInstanceRequestEthereum = newContractInstance;
 
+
+                instanceERC20TestToken.deploy({
+                    data: TestToken.bytecode,
+                    arguments: ['100000000000000000000000000000000000000000000000000000000']
+                })
+                .send({
+                    from: creator,
+                    gas: 1500000
+                }, (error: Error, transactionHash: string) => {
+                    if (error) {
+                        console.log('TestToken - error transactionHash ##########################')
+                        console.log(error)
+                        console.log(transactionHash)
+                        console.log('TestToken - error transactionHash ##########################')
+                    }
+                    // console.log('RequestCore - transactionHash : '+transactionHash);
+                })
+                .on('error', (error: Error) => {
+                    console.log('TestToken - error transactionHash ##########################')
```

Comment:
I don't understand the error message. Why is there "error transactionHash"?

Reply:
was a copy/past, I will change it